cockroachcloud/sessions-page.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Sessions Page
+summary: The Sessions page provides details of all open sessions in the cluster.
+toc: true
+---
+
+The **Sessions** page of the CockroachCloud Console provides details of all open sessions in the cluster.
+
+To view this page, click **Sessions** in the left-hand navigation of the CockroachCloud Console.
+
+## Sessions list
+
+Use the **Sessions** list to see the open sessions in the cluster. This includes active and idle sessions.
+
+{{site.data.alerts.callout_info}}
+A session is *active* if it has an open transaction (including implicit transactions, which are individual SQL statements), and *idle* if it has no open transaction. Active sessions consume hardware resources.
+{{site.data.alerts.end}}
+
+- If an active session also has an open transaction, the most recent SQL statement will be displayed in the **Statement** column.
+- To view [details of a session](#session-details), click on the **Session Age**.
```

Comment:
```suggestion
- To view [details of a session](#session-details), click the **Session Age**.
```

I don't think we have "click" vs "click on" in our style guide but the former seems a lot more common. Your call, since we don't have a style at the moment.